pkg/app/mutate_app.go

```diff
@@ -193,6 +193,11 @@ func (m *Mutator) mutateConfig(ctx context.Context, app v1alpha1.App) ([]mutator
 		return nil, nil
 	}
 
+	// Return early if app is a Management Cluster app.
```

Comment:
Can we check this first? We didn'tt need to fetch configmaps in this case.

Reply:
Definitely I missed that. Thx!